testing/bdd/features/deduplicator/1Sty1Reg4DupAtts.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?>
+<tt:tt ebuttp:sequenceIdentifier="ReSequenced1" ebuttp:sequenceNumber="12" ttp:cellResolution="32 15" ttp:clockMode="local" ttp:timeBase="clock" xml:lang="en-GB" xmlns:ebuttp="urn:ebu:tt:parameters" xmlns:ebutts="urn:ebu:tt:style" xmlns:tt="http://www.w3.org/ns/ttml" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:xml="http://www.w3.org/XML/1998/namespace">
+  <tt:head>
+    <tt:styling>
+      <!-- Should be replaced with one attribute -->
+      <tt:style xml:id="SEQ1.defaultStyle1" tts:fontFamily="sansSerif " /> 
```

Comment:
The style attribute references are to 58, 59 and 60, not 1. I don't think this file will validate.